examples/react/start-basic-static/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tanstack-start-example-basic-static",
```

Comment:
Keeping this here as a reminder that we should duplicate this example into the `e2e/start` directory and run the end-to-end test on the built output.

Reply:
Good idea